### ISSUE TITLE
parsnp -h should exit successfully

### ISSUE THREAD

**Issue by sjackman** [marbl/parsnp#4]
It exits with exit status 2. It should exit with 0.

```
❯❯❯ parsnp_OSX64_v1.0.1/parsnp -h >/dev/null; echo $?
|--Parsnp v1.0--|
For detailed documentation please see --> http://harvest.readthedocs.org/en/latest
2
```

**Comment by tseemann** [comment]
I agree. This is important in pipelines etc.

**Comment by treangen** [comment]
+1, and thanks Shaun for pointing this out, this has been fixed in v1.1:

https://github.com/marbl/parsnp/releases

**Comment by sjackman** [comment]
Thanks!
